src/gui_common/MainMenu.cs

```diff
@@ -133,6 +138,9 @@ private void RunMenuSetup()
         // Easter egg message
         ToolTipHelper.RegisterToolTipForControl(
             thriveLogo, toolTipCallbacks, ToolTipManager.Instance.GetToolTip("thriveLogoEasterEgg", "mainMenu"));
+
+        if ((int)OS.GetCurrentVideoDriver() == 1)
```

Comment:
Don't cast to int, compare the enum values directly.

Reply:
Also this code shouldn't retrigger when returning from the game so also check `IsReturningToMenu` here.